app/controllers/application_controller.rb

```diff
@@ -1,2 +1,29 @@
 class ApplicationController < ActionController::Base
+
+  def index
+    render file: "#{Rails.root}/public/about.html"
+  end
```

Comment:
It's better keep application_controller without actions(just for methods, helpers or callback) and if you want add public page, you should create new controller, for example main_page_controller and move public actions with public pages there

Reply:
Ok, it will be fixed in the next pull request